phantom/utils/__init__.py

```diff
@@ -158,6 +159,18 @@ def check_env_config(env_config: Mapping[str, Any]) -> None:
                 raise Exception
 
 
+def validate_env(env) -> None:
+    with telemetry.logger.pause():
+        obs, _ = env.reset()
+
+        for _ in range(env.num_steps):
```

Comment:
do you need to go over all the steps? can you just do one step or just a couple?

Reply:
I think it would be hard to choose a number. For standard environments a few steps would be fine but for FSM environments you are going to want to make sure you cycle through all the states.

This won't add much overhead in the scheme of the overall training/rollout process.